src/effects/VST3/VST3OptionsDialog.cpp

```diff
@@ -0,0 +1,135 @@
+/**********************************************************************
+
+  Audacity: A Digital Audio Editor
+
+  @file VST3OptionsDialog.cpp
+
+  @brief Part of Audacity VST3 module, duplicate from VSTEffect.cpp 
```

Comment:
Can the duplication be eliminated?  Can this file be shared with the old VST implementation?

Reply:
You expect that list of options will stay same for both VST and VST3?